pytket_qir/qir.py

```diff
@@ -228,6 +242,29 @@ def block_to_circuit(self, block: QirBlock, circuit: Circuit) -> Circuit:
                 circuit.add_gate(op, unitids)
             elif instr.instr.is_qir_call:  # Setting the conditional bit for branching.
                 c_reg_index = instr.instr.call_func_params[0].constant.result_static_id
+            elif instr.instr.is_call:  # WASM external call.
+                if not self.wasm_handler:
```

Comment:
```suggestion
                if self.wasm_handler is not None:
```
It's usually better to be explicit for this kind of thing: `if not x` may evaluate to false for all sorts of cases that you may or may not want to include: empty strings, empty sets, etc, as well as being user-definable. This avoids any ambiguity.

Reply:
Makes a lot of sense. Thanks.